agent/src/agent/pipeline/config/jython_scripts/jdbc.py

```diff
@@ -52,6 +83,10 @@ def main():
             time.sleep(2)
             if sdc.isStopped():
                 return
+
+        if sdc.userParams['TRACK_MISSED_INTERVALS'] == 'True':
```

Comment:
in the case of a 1-day interval missing data will be only checked once a day, but a more efficient way would be querying missing data every hour in this case, because the data may just arrive a bit later than usual. We can add a `query_missing_data_interval` config for example